bigtable/admin.go

```diff
@@ -255,12 +272,14 @@ type TableConf struct {
 	// set to protected to make the table protected against data loss
 	DeletionProtection    DeletionProtection
 	ChangeStreamRetention ChangeStreamRetention
+	// Configure an automated backup policy for the table
+	AutomatedBackupPolicy *AutomatedBackupPolicy
 }
```

Comment:
The new field should be 	
```
AutomatedBackupConfig *TableAutomatedBackupConfig
```
similar to 

https://github.com/googleapis/go-genproto/blob/7cd4c1c1f9ece082e88635ff81f99573467b5edd/googleapis/bigtable/admin/v2/table.pb.go#L684-L687

This is because automated_backup_config is a oneof field of the Table message. Currently, this field has only one sub field i.e. automated_backup_policy. This can change in the future. So, create an interface ‘AutomatedBackupConfig’ to represent automated backup configuration

https://github.com/googleapis/googleapis/blob/587784db5e4977ed5b195e15f17dddf09d5b3808/google/bigtable/admin/v2/table.proto#L186-L190

Reply:
Thank you! Updated

Reply:
Thank you! Updated